client/my-sites/email/email-management/home/utils.js

```diff
@@ -98,11 +98,23 @@ export function hasEmailSubscription( domain ) {
 	return !! subscriptionId;
 }
 
-export function resolveEmailPlanStatus( domain ) {
+/**
+ * Determines if any warnings exists with the slug `unused_mailboxes` in an array of warning objects
+ *
+ * @param {Array} warnings - An array of warning objects
+ * @returns {boolean} true if unused mailboxes exists, false otherwise
+ */
+function hasUnusedMailboxWarnings( warnings ) {
+	return ( warnings?.length ? warnings : [] ).some(
+		( warning ) => EMAIL_WARNING_SLUG_UNUSED_MAILBOXES === warning?.warning_slug
+	);
+}
+
+export function resolveEmailPlanStatus( domain, emailAccount, isLoadingEmails ) {
 	const defaultActiveStatus = {
 		statusClass: 'success',
-		icon: 'check_circle',
-		text: translate( 'Active' ),
+		icon: isLoadingEmails ? 'cached' : 'check_circle',
+		text: isLoadingEmails ? translate( 'Loading accounts' ) : translate( 'Active' ),
```

Comment:
Nit: I think this may be simpler to express as "Loading details" -- the user doesn't know how an account maps to the screen.